src/PurpleBooth/GitLintValidators/Composer/Scripts.php

```diff
@@ -85,4 +100,54 @@ public static function installGitMessageHook(Event $event)
 
         return false;
     }
+
+    /**
+     * Installs and configures the Git commit message
+     * template when confirmed by the user.
+     *
+     * @param  Event $event The script event.
+     *
+     * @return boolean
+     */
+    public static function installGitCommitMessageTemplate(Event $event)
+    {
+        $templateContent = self::TEMPLATE_CONTENTS;
+        $inputOutput = $event->getIO();
+        $question    = "Do you want to install and configure the Git ";
+        $question   .= "commit message template? ";
```

Comment:
!fix Equals sign not aligned with surrounding assignments; expected 7 spaces but found 3 spaces